### PR TITLE
fixed the public key of the alastria session object

### PR DESCRIPTION
**Pull request by rober12**

### CODE REVIEW

File: exampleAuthentication/1.authentication.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ console.log("entity1Puk pubk:", '04'+entity1Puk.getPublicKeyString().substr(2))
   console.log('\tIs the signedJWT verified?', verifyAT)
 
 
-const alastriaSession = tokensFactory.tokens.createAlastriaSession(context, configData.didSubject1, "04"+entity1Puk.getPublicKeyString().substr(2), signedAT, tokenExpTime, tokenActivationDate, jsonTokenId)
+const alastriaSession = tokensFactory.tokens.createAlastriaSession(context, configData.didSubject1, "04"+subject1Pubk.getPublicKeyString().substr(2), signedAT, tokenExpTime, tokenActivationDate, jsonTokenId)
```
Comment (VictorNS69) [inline exampleAuthentication/1.authentication.js]: I think this should be without "04" and with "0x"
```suggestion
const alastriaSession = tokensFactory.tokens.createAlastriaSession(context, configData.didSubject1, subject1Pubk.getPublicKeyString(), signedAT, tokenExpTime, tokenActivationDate, jsonTokenId)
```

**Review by VictorNS69 — CHANGES_REQUESTED** [review]

File: exampleAuthentication/1.authentication.js (hunk 1 above)
Comment (rober12) [reply exampleAuthentication/1.authentication.js]: Changed

**Review by rober12 — COMMENTED** [review]
Changed. Correctly now.

File: exampleAuthentication/1.authentication.js [hunk 2]
```diff
@@ -64,16 +64,17 @@ console.log("entity1Puk pubk:", '04'+entity1Puk.getPublicKeyString().substr(2))
   let verifyAT = tokensFactory.tokens.verifyJWT(signedAT, "04"+entity1Puk.getPublicKeyString().substr(2))
   console.log('\tIs the signedJWT verified?', verifyAT)
 
+  const subject1Puk = Wallet.fromPrivateKey(subject1PrivateKey)
+  // '04' means uncompressed key (more info at https://github.com/indutny/elliptic/issues/138)
+  console.log("entity1Puk pubk:", '04'+subject1Puk.getPublicKeyString().substr(2))
 
-const alastriaSession = tokensFactory.tokens.createAlastriaSession(context, configData.didSubject1, "04"+subject1Pubk.getPublicKeyString().substr(2), signedAT, tokenExpTime, tokenActivationDate, jsonTokenId)
+const alastriaSession = tokensFactory.tokens.createAlastriaSession(context, configData.didSubject1, subject1Puk.getPublicKeyString(), signedAT, tokenExpTime, tokenActivationDate, jsonTokenId)
```
Comment (VictorNS69) [inline exampleAuthentication/1.authentication.js]: Just take the public key from the config file 
image

**Review by VictorNS69 — CHANGES_REQUESTED** [review]

**Review by rober12 — COMMENTED** [review]
cleaned

**Review by VictorNS69 — APPROVED** [review]

**Pull request merged**